src/Identity/Port/Adapter/Persistence/Migration/Version20170526204325.php

```diff
@@ -15,12 +15,12 @@ public function up(Schema $schema): void
 
         $table->addColumn('id', 'uuid');
         $table->addColumn('version', 'integer', ['default' => 1]);
-        $table->addColumn('is_signed_up', 'boolean');
-        $table->addColumn('credentials_username', 'string', ['notNull' => false, 'length' => 255]);
-        $table->addColumn('credentials_password', 'string', ['notNull' => false, 'length' => 255]);
+        $table->addColumn('email', 'string', ['notNull' => false, 'length' => 255]);
```

Comment:
Although the theoretical limit is somewhat around `320`, a length of `128` should be good enough.